README.md

```diff
@@ -147,17 +145,12 @@ Modify `dymint.toml` in the chain directory (`~/.rollapp_evm/config`)
 linux:
```

Comment:
will this work on mac?

Reply:
sorry yes meant to delete the linux word